LibreNMS/Data/Source/SnmpResponse.php

```diff
@@ -68,7 +68,7 @@ public function isValid(): bool
     {
         $this->errorMessage = '';
         // not checking exitCode because I think it may lead to false negatives
-        $invalid = preg_match('/(Timeout: No Response from .*|Unknown user name|Authentication failure)/', $this->stderr, $errors)
+        $invalid = preg_match('/(Timeout: No Response from .*|Unknown user name|Authentication failure|Error: OID not increasing: .*)/', $this->stderr, $errors)
             || empty($this->raw)
```

Comment:
shouldnt we check this first? faster than a large regex

Reply:
@Jellyfrog had it first, but notice, "raw" is stdout, the first regex checks stderr.  This results in the wrong error message.  The program flow ends up looking like: if the output is empty check stderr regex, otherwise, check stderr regex.  You could check if stderr is empty first, but I suspect that is similar in performance as a regex against an empty string.